app/services/decorators/state_benefit_decorator.rb

```diff
@@ -0,0 +1,23 @@
+module Decorators
+  class StateBenefitDecorator
+    def initialize(state_benefit)
+      @state_benefit = state_benefit
+    end
+
+    def as_json
+      {
+        name: @state_benefit.state_benefit_name,
+        monthly_value: @state_benefit.monthly_value,
+        state_benefit_payments: payments
```

Comment:
Do we include whether or not it's excluded from the gross income summary into the json response?

Reply:
yep, I'll add this.